blocks/columns/columns.js

```diff
@@ -34,6 +34,31 @@ export function applySplitPercentages(block) {
   }
 }
 
+function embedWistia(url) {
```

Comment:
can we somehow merge these ?
https://github.com/aemsites/stericycle-shared/blob/59cef1134de57d53139485d79d5b26f1dcb417fb/blocks/video/video.js#L54